src/cmd/apply.ts

```diff
@@ -97,10 +101,11 @@ const applyAll = async () => {
     },
     { streams: { stdout: d.stream.log, stderr: d.stream.error } },
   )
-  // commit first time only if cli, always commit in chart (might have previous failure)
-  if (!env.DISABLE_SYNC && (!isCli || !status)) {
-    await commit(true)
-  }
+  if (!env.DISABLE_SYNC)
+    if (isChart || !prevState.status)
```

Comment:
Would below statement be more accurate?
```
if (isChart || prevState.status !== deployed)
```

Reply:
no, this is about existence of that data in the first place